frame/contracts-registry/src/benchmarking.rs

```diff
@@ -0,0 +1,103 @@
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+
+use frame_benchmarking::impl_benchmark_test_suite;
+use frame_benchmarking::v1::{account, benchmarks, BenchmarkError};
+use frame_support::{
+	assert_ok, ensure,
+	traits::{EnsureOrigin, Get},
+};
+use frame_system::RawOrigin;
+use sp_runtime::BoundedVec;
+use sp_std::vec;
+
+use crate::Pallet as ContractsRegistry;
+
+const SEED: u32 = 0;
+
+benchmarks! {
+	add_judge {
+		let origin =
+			T::AddJudgeOrigin::try_successful_origin().map_err(|_| BenchmarkError::Weightless)?;
+		let user: T::AccountId = account("user", 0, SEED);
+
+
+		for s in 0..T::MaxJudges::get()-1 {
+			assert_ok!(ContractsRegistry::<T>::add_judge(origin.clone(), user.clone()));
+		}
+	}: _<T::RuntimeOrigin>(origin, user)
+	verify {
+		// External proposal created
```

Comment:
`External proposal`? Is this some leftover copied from democracy :D? Cause we have no external proposals in contract-registry

Reply:
Looks like this is repeated in all benchmarks.